engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/schemaregistry/confluent/client/CachedConfluentSchemaRegistryClientFactory.scala

```diff
@@ -15,7 +15,7 @@ class CachedConfluentSchemaRegistryClientFactory(maximumSize: Long, latestSchema
   //Cache engines are shared by many of CachedConfluentSchemaRegistryClient
   @transient private lazy val caches = new SchemaRegistryCaches(maximumSize, latestSchemaExpirationTime, schemaExpirationTime, versionsCacheExpirationTime)
 
-  override def createSchemaRegistryClient(kafkaConfig: KafkaConfig): ConfluentSchemaRegistryClient = {
+  override def create(kafkaConfig: KafkaConfig): ConfluentSchemaRegistryClient = {
```

Comment:
What is the purpose of renaming this method? I would prefer to keep it named old way instead of resolve problems after code migration.

Reply:
Ok, I see now that it is for purpose of being consistent with `DeserializationSchemaFactory` and so on